github/event_types.go

```diff
@@ -653,6 +675,21 @@ type StatusEvent struct {
 	Installation *Installation     `json:"installation,omitempty"`
 }
 
+// TeamEvent is triggered when an organization's team is created, modified or deleted.
+// The Webhook event name is "team".
+//
+// Events of this type are not visible in timelines. These events are only used
+// to trigger hooks.
+//
+// GitHub API docs: https://developer.github.com/v3/activity/events/types/#teamevent
+type TeamEvent struct {
+	Action  *string       `json:"action,omitempty"`
+	Team    *Team         `json:"team,omitempty"`
+	Changes *TeamChange   `json:"changes,omitempty"`
+	Org     *Organization `json:"organization,omitempty"`
+	Sender  *User         `json:"sender,omitempty"`
```

Comment:
Finally, I think it'd be nice to separate the webhook-only fields from the events API ones, like this:

```Go
type TeamEvent struct {
	...

	// The following fields are only populated by Webhook events.
	Org     *Organization `json:"organization,omitempty"`
	Sender  *User         `json:"sender,omitempty"`
	Installation *Installation `json:"installation,omitempty"`
}
```

You can see the `TeamAddEvent` below for example.